lib/datasource/pod/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable typescript-enum/no-enum */
```

Comment:
Or how to disable two rules at once for a single line?

Reply:
Use a comma separated list, see other comment